src/coreclr/inc/corinfo.h

```diff
@@ -1970,6 +1931,23 @@ struct CORINFO_SWIFT_LOWERING
     size_t numLoweredElements;
 };
 
+#define MAX_FPSTRUCT_LOWERED_ELEMENTS 2
+
+// Lowering information on fields of a struct passed by hardware floating-point calling convention on RISC-V and LoongArch
+// NOTE: all array elements past numLoweredElements are zero-initialized
```

Comment:
Does anything on the JIT side depend on this?

We tend to avoid zero-initializing unused space in the JIT/EE structs. It is unnecessary overhead as the structs grow larger.

Reply:
Not really, I assumed `(numLoweredElements == 0) == byIntegerCallConv` to make the code cleaner (as did CORINFO_SWIFT_LOWERING in one place). Added checks.